packages/core/src/lib/keep_alive_manager.ts

```diff
@@ -1,12 +1,11 @@
 import type { PeerId } from "@libp2p/interface-peer-id";
 import type { IRelay } from "@waku/interfaces";
+import { RelayPingContentTopic } from "@waku/utils/constants";
```

Comment:
I am not convinced constants need to all be in @waku/utils. This seems like a constant that should go in @waku/relay.

Probably can remain in core for now.